app/cdap/components/DataPrep/TopPanel/index.js

```diff
@@ -190,6 +190,7 @@ export default class DataPrepTopPanel extends Component {
   renderTopPanelDisplay() {
     const { connectionName, path, workspaceName } = this.state;
     let connectionInfo = `${connectionName} - ${path}`;
+    let state = DataPrepStore.getState().dataprep;
```

Comment:
A couple small improvements here:
1. Rename the variable to `dataprep` to avoid any confusion with React state?
2. Use `const`?

Then this line can be

```
const { dataprep } = DataPrepStore.getState();
```

Reply:
Thanks, Nick will make the changes and create a new pull request.